lib/db/icv.ts

```diff
@@ -0,0 +1,190 @@
+/**
+ * @module stardogjs.db.icv
+ */
+import { BaseDatabaseOptions, BaseDatabaseOptionsWithGraphUri } from '../types';
+import { RequestHeader, ContentType, RequestMethod } from '../constants';
+import { getFetchDispatcher } from '../request-utils';
+
+const dispatchFetchWithGraphUri = getFetchDispatcher({
+  allowedQueryParams: ['graph-uri'],
+});
```

Comment:
I've seen this method a couple times now (on graph.ts too, for instance). Would it make any sense to move it to request-utils and import as needed?